dp-terraform/helm/rhacs-terraform/Chart.yaml

```
@@ -38,3 +38,6 @@ dependencies:
     version: "0.21.1"
     repository: "https://helm.vector.dev"
     condition: vector.enabled
+  - name: secured-cluster
+    version: "0.1.0"
+    condition: securedCluster.enabled
```

```suggestion
    condition: secured-cluster.enabled
```
Need to double-check whether it works